pytest.ini.example

```diff
@@ -19,9 +19,6 @@ env =
     azext_iot_testdps=
     azext_iot_teststorageuri=
     azext_iot_testidentity=
-    azext_pnp_endpoint=
-    azext_pnp_repository=
```

Comment:
NICE!!!

Reply:
Are there any new vars that we need in ini for the new IT's?

Reply:
I know right! Main thing we need to do for IT's is to configure the IT SP with rights on the tenant repository